src/Shared/TaskLoader.cs

```diff
@@ -179,11 +177,11 @@ bool isOutOfProc
         /// This is a resolver to help created AppDomains when they are unable to load an assembly into their domain we will help
         /// them succeed by providing the already loaded one in the currentdomain so that they can derive AssemblyName info from it
         /// </summary>
-        internal static Assembly AssemblyResolver(object sender, ResolveEventArgs args)
+        internal static Assembly? AssemblyResolver(object sender, ResolveEventArgs args)
         {
-            if (args.Name.Equals(s_resolverLoadedType.LoadedAssemblyName.FullName, StringComparison.OrdinalIgnoreCase))
+            if (args.Name.Equals(s_resolverLoadedType?.LoadedAssemblyName.FullName, StringComparison.OrdinalIgnoreCase))
             {
-                return s_resolverLoadedType.LoadedAssembly ?? Assembly.Load(s_resolverLoadedType.Path);
+                return s_resolverLoadedType?.LoadedAssembly ?? Assembly.Load(s_resolverLoadedType?.Path);
```

Comment:
```suggestion
                return s_resolverLoadedType?.LoadedAssembly ?? Assembly.Load(s_resolverLoadedType!.Path);
```
the if check above ensures it's not null right?

Reply:
Yes, I think so